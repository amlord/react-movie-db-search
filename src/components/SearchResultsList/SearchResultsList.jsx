import React from 'react';
// import styled from 'styled-components';

// const StyledSearchBar = styled.div`
//   background-color: #ececed;
//   color: #333;
// `;

const SearchResultsList = ({ results }) => (
  <>
    {results.map(result => <div key={result.id}>{result.title}</div>)}
  </>
);

export default SearchResultsList;
