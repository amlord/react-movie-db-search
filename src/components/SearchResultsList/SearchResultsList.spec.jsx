import React from 'react';
import { shallow } from 'enzyme';
import SearchResultsList from './SearchResultsList';

describe('SearchBar', () => {
  it('displays correctly', () => {
    const wrapper = shallow(<SearchResultsList results={[]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
