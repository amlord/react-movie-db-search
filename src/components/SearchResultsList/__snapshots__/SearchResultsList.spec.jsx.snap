// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar displays correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SearchResultsList
    results={Array []}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [],
    },
    "ref": null,
    "rendered": Array [],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [],
      },
      "ref": null,
      "rendered": Array [],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
